src/dcqc/tests/tests.py

```diff
@@ -36,13 +37,62 @@ def compute_status(self) -> TestStatus:
     def _compute_md5_checksum(self, file: File) -> str:
         local_path = file.get_local_path()
         hash_md5 = hashlib.md5()
-        with local_path.open("rb") as f:
-            for chunk in iter(lambda: f.read(4096), b""):
+        with local_path.open("rb") as infile:
+            for chunk in iter(lambda: infile.read(4096), b""):
                 hash_md5.update(chunk)
         actual_md5 = hash_md5.hexdigest()
         return actual_md5
 
 
+class JsonLoadTest(TestABC):
+    tier = 2
+    only_one_file_targets = False
+
+    def compute_status(self) -> TestStatus:
+        status = TestStatus.PASS
+        for file in self.target.files:
+            if not self._can_be_loaded(file):
+                status = TestStatus.FAIL
```

Comment:
Do you think the `_can_be_loaded` test be implemented for all file types?

Reply:
Once there is a third internal load test that would use the same pattern, I'll add a helper method that can be used by all three use cases. I might use the [template method](https://refactoring.guru/design-patterns/template-method) design pattern.